data/npclib/npc_system/bank_system.lua

```diff
@@ -180,19 +180,41 @@ function Npc:parseBank(message, npc, creature, npcHandler)
 		return true
 	elseif npcHandler:getTopic(playerId) == 7 then
 		if MsgContains(message, "yes") then
+			local totalValue = count[playerId]
+			local crystalCoins = math.floor(totalValue / 10000)
+			totalValue = totalValue % 10000
+			local platinumCoins = math.floor(totalValue / 100)
+			totalValue = totalValue % 100
+			local goldCoins = math.floor(totalValue / 1)
+			local crystalPiles = math.floor((crystalCoins + 99) / 100)
+			local platinumPiles = math.floor((platinumCoins + 99) / 100)
+			local goldPiles = math.floor((goldCoins + 99) / 100)
+			local totalPiles = crystalPiles + platinumPiles + goldPiles
```

Comment:
These local variables cannot be placed inside ELSE? as they will only be used there.

Reply:
We need to use them to calculate totalPiles